src/app/_utils/cmsFieldUtils.ts

```diff
@@ -0,0 +1,34 @@
+import { type CMSCollectionName, type CMSFieldOption } from '@/types/cmsConfig'
+
+import { getCMSFieldOptions, getCollectionConfig } from '@/utils/cmsConfigUtils'
+
+function formatFieldOptions(options: Array<CMSFieldOption>) {
+  return options.map((option) => ({
+    id: option.value,
+    name: option.label,
+  }))
+}
+
+/**
+ * Retrieves and formats field settings for a specified collection and field.
+ *
+ * @param {CMSCollectionName} collectionName - The name of the CMS collection.
+ * @param {string} fieldName - The name of the field within the collection.
+ * @returns {{ options: { id: string; name: string }[]; validIds: string[] }}
+ * An object containing formatted options for the field and a list of valid IDs.
+ */
```

Comment:
We avoid comments in code for two reasons: they often become outdated during maintenance, and well-written functions should be self-documenting through explicit naming and structure.